<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterImageDlg</class>
 <widget class="QDialog" name="RegisterImageDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelImageName">
   <property name="geometry">
    <rect>
     <x>705</x>
     <y>47</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 140, 140);
background-color: rgb(191, 191, 191);</string>
   </property>
   <property name="text">
    <string>基准图像</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>570</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>图像名称</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRegister">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>560</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDel">
   <property name="geometry">
    <rect>
     <x>711</x>
     <y>699</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQuit">
   <property name="geometry">
    <rect>
     <x>881</x>
     <y>699</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetImage">
   <property name="geometry">
    <rect>
     <x>705</x>
     <y>76</y>
     <width>261</width>
     <height>601</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="resizeMode">
    <enum>QListView::Fixed</enum>
   </property>
   <property name="viewMode">
    <enum>QListView::ListMode</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelImage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 50, 50);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>注册图像</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtImageName">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>570</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>796</x>
     <y>699</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonSmall">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>50</y>
     <width>55</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>小图</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonBig">
   <property name="geometry">
    <rect>
     <x>809</x>
     <y>50</y>
     <width>55</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>大图</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
